<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>270</width>
    <height>510</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="gb_serial">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Serial Port</string>
    </property>
    <widget class="QPushButton" name="pb_serial_oepn">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_serial_close">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_serial_ports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_serial_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_control">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>Control</string>
    </property>
    <widget class="QLineEdit" name="le_lp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QCheckBox" name="cb_sensor_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Sensor</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_gate_open">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD_GATE_OPEN</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_gate_close">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>50</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD_GATE_CLOSE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_gate_ignore">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>80</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD_GATE_IGNORE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_display_lp">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>121</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>CMD_DISPLAY_LP</string>
     </property>
    </widget>
   </widget>
   <widget class="QPlainTextEdit" name="pte_terminal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>251</width>
      <height>241</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="lineWrapMode">
     <enum>QPlainTextEdit::NoWrap</enum>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
